lib/agent.js

```diff
@@ -350,17 +351,28 @@ Agent.prototype.start = function start(callback) {
 
     this.setState('errored')
     sampler.stop()
-    const self = this
     return process.nextTick(function onNextTick() {
-      self.healthReporter.stop(() => {
+      agent.healthReporter.stop(() => {
         callback(new Error('Not starting without license key!'))
       })
     })
+  } else if (validateLicenseKey(this.config.license_key) === false) {
```

Comment:
this state is meant for when the collector responds with a 401.  The code is [here](https://github.com/newrelic/node-newrelic/blob/main/lib/collector/api.js#L236)

```
NR-APM-001 | Invalid license key (HTTP status code 401)
```

Reply:
we should also be setting the status for the [branch above](https://github.com/newrelic/node-newrelic/blob/main/lib/collector/api.js#L233) which is a 410

```
NR-APM-003 | Forced disconnect received from New Relic (HTTP status code 410)
```